src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorWorkflowStep.groovy

```diff
@@ -67,11 +67,20 @@ class IqPolicyEvaluatorWorkflowStep
     this.jobCredentialsId = jobCredentialsId
   }
 
+  @DataBoundSetter
+  public void setIqApplication(final String iqApplication) {
```

Comment:
Does it work to have two `DataBoundSetter` for the same property? IIRC when looking at the Jenkins code only one of these is resolved.

Reply:
> Does it work to have two DataBoundSetter for the same property? IIRC when looking at the Jenkins code only one of these is resolved.

No does not work.  Went back to Object.  Couldn't use instanceof, not allowed by code quality check. 

changed df803ed

Reply:
>Couldn't use instanceof, not allowed by code quality check.

Changed 83aa26a